docs/pages/index.mdx

```diff
@@ -40,6 +39,23 @@ slug: wallets
   </Button>
 </div>
 
+<style>{`
+  :root {
+    --react-quickstart-btn-bg: var(--accent-2, #ECF3FF);
+    --react-quickstart-btn-color: black;
+  }
+
+  html[data-theme='dark'] {
+    --react-quickstart-btn-bg: #777777;
+    --react-quickstart-btn-color: white;
+  }
+
+  .react-quickstart-btn {
+    background-color: var(--react-quickstart-btn-bg);
+    color: var(--react-quickstart-btn-color);
+  }
```

Comment:
Fern supports dark mode out of the box right? I think this change would be better defined at the design system level as opposed to a one-off style on the button